utils/update_assignment_problems.py

```diff
@@ -67,8 +67,8 @@ def handle_input() -> Tuple[str, str, str]:
     return args.api_token, args.url, args.input
 
 
-def assignment_exists(assignments: List[Dict[str, Any]], alias: str) -> bool:
-    return any(a["alias"] == alias for a in assignments)
+def assignment_exists(assignments: List[Any], alias: str) -> bool:
```

Comment:
I don’t really like the use of the `Any` type, since this clearly seems to be an object. 

But to speed up the review, we can fix that later by implementing the `mypy` type checker (in any case, that’s out of the scope of this project).